### ISSUE TITLE
Level A Validation Error - "Error: Duplicate attribute class" on /people page using PP1

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#827]
# Who does this affect?
Anonymous users

# What is occurring?
For a page using the PP1 view associated with the URL /people, I'm seeing a validation issue: ```Error: Duplicate attribute class.``` 

This causes the page to fail under [WCAG 2.0 Level A Success Criterion 4.1.1. Parsing](https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=411#qr-ensure-compat-parses)

# Steps to Reproduce
1. Ensure the UG Profile Layouts feature is in a default state
2. Ensure PP1 view pane is being used on /people
3. Note that the default settings for the view-pane appeared as follows: title was overridden as blank, provide a 'more' link is unchecked, override "more" text is blank, pager ID is checked, Use pager is set to 0 and Num items is set to 10.
![image](https://user-images.githubusercontent.com/1927902/30713310-e85fdaa0-9edc-11e7-9b97-e32cd1e41a0f.png)
4. Run the page through the [W3C HTML validator](https://validator.w3.org/)

According to the HTML validator, it's finding code snippets of the format: ```<article class="media"  class="node node-profile node-teaser view-mode-teaser">```. The duplicate class is the reason for the validation error.

Note: When I review the code in the Web Inspector, the code shows ```<article class="media">``` and I do not see ```class="node node-profile node-teaser view-mode-teaser"```. However it does show up when I view the page source or use the validator (i.e. not the DOM).

# Why does it matter?
Any sites using a default version of the UG Profile Layouts feature will automatically have this validation error if it uses the /people page.

# How to fix?
After doing a validation check on PP6, it looks like the PP6 view pane does not exhibit the same validation issues. 

Therefore, you can do one or both of the following:
1. Option A - Fix the PP1 view so it does not cause this error (possibly using PP6 as a guide).
2. Option B - Update the /people page layout, so by default, it uses the PP6 view pane (with the following settings: small image turned off, align images turned on) instead of the PP1 view pane.

Note that once the code is fixed, you will still likely have to update sites on a case-by-case basis since some sites may be using overridden UG People Profile and UG Profile Layout features. This would either involve reverting the profile features or ensuring that the sites use the PP6 view pane instead of the PP1 view pane.

**Comment by nevantan** [comment]
The validation issue is due to something in the Bootstrap theme. it doesn't occur in PP6 because we've overridden the Bootstrap template for other reasons. I think option 2 is likely the best way to go.
